### PR TITLE
Update TTS task page

### PR DESCRIPTION
**Pull request by Deep-unlearning**

### CODE REVIEW

**Comment by Wauplin** [comment]
(gently pinging @merveenoyan as well on this one)

**Review by pcuenca — APPROVED** [review]
Looks good to me, assuming it's ready for review :)

**Review by Vaibhavs10 — APPROVED** [review]
Nice! Thanks for updating this @Deep-unlearning 🔥 

(looked at this offline w/ Steven)

File: packages/tasks/src/tasks/text-to-speech/data.ts [hunk 1]
```diff
@@ -66,8 +74,8 @@ const taskData: TaskDataCustom = {
 			id: "mrfakename/E2-F5-TTS",
 		},
 		{
-			description: "An application that synthesizes speech for diverse speaker prompts.",
-			id: "parler-tts/parler_tts_mini",
+			description: "An application that synthesizes speech in a given emotion and voice that can be controlled through a simple text prompt.",
```
Comment (Vaibhavs10) [inline packages/tasks/src/tasks/text-to-speech/data.ts:77]:
```suggestion
			description: "An application that synthesizes emotional speech for diverse speaker prompts.",
```

**Pull request merged**